lib/fathead/perl_doc/parse.sh

```diff
@@ -1 +1,2 @@
 rm -f {Parsers/,}output.txt && cd Parsers && perl parse.pl && mv output.txt ..
+LC_ALL=C sort output.txt -o output.txt
```

Comment:
Fixed.

@GuiltyDolphin FYI, when I ran the parse I saw a bunch of `uninitialized` warnings:

```
Use of uninitialized value $modifier in substitution (s///) at parse.pl line 664.
Use of uninitialized value $modifier in concatenation (.) or string at parse.pl line 665.
Use of uninitialized value $modifier in concatenation (.) or string at parse.pl line 665.
Use of uninitialized value $modifier in concatenation (.) or string at parse.pl line 665.
Use of uninitialized value $modifier in substitution (s///) at parse.pl line 664.
Use of uninitialized value $modifier in concatenation (.) or string at parse.pl line 665.
Use of uninitialized value $modifier in concatenation (.) or string at parse.pl line 665.
Use of uninitialized value $modifier in concatenation (.) or string at parse.pl line 665.
Use of uninitialized value $modifier in substitution (s///) at parse.pl line 664.
Use of uninitialized value $modifier in concatenation (.) or string at parse.pl line 665.
Use of uninitialized value $modifier in concatenation (.) or string at parse.pl line 665.
Use of uninitialized value $modifier in concatenation (.) or string at parse.pl line 665.
Use of uninitialized value $modifier in substitution (s///) at parse.pl line 664.
Use of uninitialized value $modifier in concatenation (.) or string at parse.pl line 665.
Use of uninitialized value $modifier in concatenation (.) or string at parse.pl line 665.
Use of uninitialized value $modifier in concatenation (.) or string at parse.pl line 665.
Use of uninitialized value $modifier in substitution (s///) at parse.pl line 664.
Use of uninitialized value $modifier in concatenation (.) or string at parse.pl line 665.
Use of uninitialized value $modifier in concatenation (.) or string at parse.pl line 665.
Use of uninitialized value $modifier in concatenation (.) or string at parse.pl line 665.
Use of uninitialized value $modifier in substitution (s///) at parse.pl line 664.
Use of uninitialized value $modifier in concatenation (.) or string at parse.pl line 665.
Use of uninitialized value $modifier in concatenation (.) or string at parse.pl line 665.
Use of uninitialized value $modifier in concatenation (.) or string at parse.pl line 665.
Use of uninitialized value $modifier in substitution (s///) at parse.pl line 664.
Use of uninitialized value $modifier in concatenation (.) or string at parse.pl line 665.
Use of uninitialized value $modifier in concatenation (.) or string at parse.pl line 665.
Use of uninitialized value $modifier in concatenation (.) or string at parse.pl line 665.
```

Thanks to the sorting I immediately noticed the first 27 lines of the `output.txt` were blank, the next 134 were just lines containing 10 tab characters, after that there are hundreds of articles that don't have a title, and hundreds of redirects that don't have a title, hundreds more blank lines. It looks like the first correct entry in the output is line 1704 ☹️ 

I'm hoping the output is just messed up because of those uninitialized errors?